### ISSUE TITLE
Combat abilities

### ISSUE THREAD

**Issue by Gedan** [Gedan/the-silence#8]
It took a few tries (it wasn’t obvious that the Palaneen (I forget his name) got his throwing axes back after each fight) but I eventually got the last boss down to the point where she restores her hp and shields.
Considering you’re not supposed to win the fight, I’d count reaching that trigger as victory. I think once healing is enabled between battles, it’ll become much easier. Healing cloud also acting as a revive was neat, but IDK if it’s a bug.

Otherwise, Tarik’s cleave attack still targets downed FOEs and it might be because he drew a paralysis (It hit security droid A for a full 1 damage after it’d had it’s health down to zero),
however after using his ‘battlecry’ special and being subsequently knocked out for it’s duration, even after I had the message saying it had worn off, it still shows “focus fire [-2 rounds]” under misc. (with -2 decrementing each turn)

tl;dr, last-round actions should be getting cached, and one of the things I was gunning for was just repeating the last action-set if you didn’t select anything. I probably broke it fixing something else just before pushing a release ;-;
